### PR TITLE
Make Modal close on Enter and Escape keys press

### PR DESCRIPTION
**Pull request by sradevski**
Connects-to: #763
Change-type: minor
Signed-off-by: Stevche Radevski <stevche@balena.io>

### CODE REVIEW

File: src/components/Modal.tsx [hunk 1]
```diff
@@ -88,8 +90,32 @@ class Modal extends React.Component<ThemedModalProps, any> {
...
 		}
+
+		window.document.removeEventListener('keydown', this.handleKeyDown);
 	}
 
+	handleKeyDown = (e: KeyboardEvent) => {
+		if (this.props.keyDownHandlers && this.props.keyDownHandlers[e.which]) {
+			this.props.keyDownHandlers[e.which](e);
+		} else {
+			console.log(e.defaultPrevented);
```
Comment (karaxuna) [inline src/components/Modal.tsx]: Forgot to remove this.

**Comment by LucianBuzzo** [comment]
@sradevski Why is there a `keyDownHandlers` prop?

**Comment by sradevski** [comment]
@LucianBuzzo In case you want to override the default behavior (not close the dialog on escape, not submit on enter), or bind other keys with different behavior.

**Comment by LucianBuzzo** [comment]
@sradevski Is this a requirement right now? I don't think that being able to override the default behaviour is a good thing

**Comment by sradevski** [comment]
@LucianBuzzo I just wanted to provide an escape hatch in case there is a specific place where the default behavior is not desired. Anyhow, I agree we can leave this for later on if required and I can remove it from this PR.

**Comment by sradevski** [comment]
@thgreasi I did, but then I did another change that broke input! Nice catch!

**Comment by sradevski** [comment]
@resin-ci retest

**Pull request merged**